templates/helm-defaults.yml

```diff
@@ -5,7 +5,10 @@ generic-service:
   resources:
     requests:
       cpu: 128m
-      memory: 512Mi
+      memory: 1024Mi
```

Comment:
Is this a bit high, given we're averaging under 700 in prod?
```
$ kubectl -n hmpps-probation-integration-services-prod top pod | awk '{sum+=$3} END {print "Avg: " sum/NR}'
Avg: 695.882
```

Maybe 768Mi would be more appropriate?